lhotse/features/mixer.py

```diff
@@ -50,7 +50,11 @@ def __init__(
         self.num_channels = 1 if base_feats.ndim == 2 else base_feats.shape[-1]
         self.gains = []
         self.frame_shift = frame_shift
-        self.padding_value = padding_value
+        self.padding_value = (
+            feature_extractor.padding_value
+            if hasattr(feature_extractor, "padding_value")
```

Comment:
I think since this behavior is undocumented in this PRs current shape, it may lead to some surprises / be hard to discover for implementers of other feature extractors how to handle this. What if we implemented this property on the base feature extractor class and made it return `None` there, and check for None here instead? Then we can add a proper docstring in feature extractor / documentation.